api/modules/publishedProjects/model.js

```diff
@@ -0,0 +1,31 @@
+var BaseModel = require('../../classes/base_model');
+
+var Projects = require('../projects/model');
+
+var instanceProps = {
+  tableName: 'publishedProject',
```

Comment:
Correct - we can change this if you want. The reason that's a convention is because sql has traditionally been really unreliable with case sensitivity. In postgres, it will respect it.

Your call!